docs/getting_started/benchmark_owner_demo.md

```diff
@@ -228,15 +226,15 @@ Now the workspace should look like this:
     ...
 ```
 
-Finally, compress the required assets (`mock_chexpert` and `paths.yaml`) into a tarball file. In your workspace directory, run:
+Finally, compress the required assets (`mock_chexpert` and `paths.yaml`) into a tarball file and then run the following command in your workspace directory:
```

Comment:
Finally, compress the required assets (`mock_chexpert` and `paths.yaml`) into a tarball file `by running the` following command in your workspace directory: